boards/arm/bbc_microbit/bbc_microbit.dts

```diff
@@ -1,77 +1,89 @@
-/*
- * Copyright (c) 2017 Linaro Limited
- *
- * SPDX-License-Identifier: Apache-2.0
- */
-
-/dts-v1/;
-#include <nordic/nrf51822_qfaa.dtsi>
-
-/ {
-	model = "BBC Micro:bit";
-	compatible = "bbc,microbit", "nordic,nrf51822-qfaa", "nordic,nrf51822";
-
-	chosen {
-		zephyr,console = &uart0;
-		zephyr,sram = &sram0;
-		zephyr,flash = &flash0;
-	};
-};
-
-&gpiote {
-	status ="ok";
-};
-
-&gpio0 {
-	status ="ok";
-};
-
-&uart0 {
-	status = "ok";
-	current-speed = <115200>;
-	tx-pin = <24>;
-	rx-pin = <25>;
-};
-
-&i2c0 {
-	status = "ok";
-	clock-frequency = <I2C_BITRATE_FAST>;
-	sda-pin = <30>;
-	scl-pin = <0>;
-};
-
-&flash0 {
-	/*
-	 * For more information, see:
-	 * http://docs.zephyrproject.org/latest/devices/dts/flash_partitions.html
-	 */
-	partitions {
-		compatible = "fixed-partitions";
-		#address-cells = <1>;
-		#size-cells = <1>;
-
-		boot_partition: partition@0 {
-			label = "mcuboot";
-			reg = <0x00000000 0x8000>;
-		};
-		slot0_partition: partition@8000 {
-			label = "image-0";
-			reg = <0x00008000 0x1a000>;
-		};
-		slot1_partition: partition@22000 {
-			label = "image-1";
-			reg = <0x00022000 0x1a000>;
-		};
-		scratch_partition: partition@3c000 {
-			label = "image-scratch";
-			reg = <0x0003c000 0x2000>;
-		};
-
-#if defined(CONFIG_FS_FLASH_STORAGE_PARTITION)
-		storage_partition: partition@3e000 {
-			label = "storage";
-			reg = <0x0003e000 0x00002000>;
-		};
-#endif
-	};
-};
```

Comment:
Why are there so many changes in this file? Did you convert line endings? I'd expect the only change to be adding the sensor node to the i2c bus.

Reply:
I'm not sure if the i2c config was correct, looking for advice.